app/enterprise/1.5.x/developer-portal/theme-customization/emails.md

```diff
@@ -12,11 +12,11 @@ Editable email templates are loaded as files similar to content files for portal
 Email files can be managed in the same way as other files for rendering, via editor or via the Portal CLI Tool.
 This feature is **not** supported on legacy portal mode.
 
-If no email templates are loaded, Kong will fall back to the same emails as Kong Enterprise 1.3.0.0
-By default on 1.3.0.1 and newer enabling a non-legacy portal on new workspaces loads default editable email templates.
+If no email templates are loaded, Kong will fall back to the same emails as Kong Enterprise 1.3.0.0.
+By default on 1.3.0.1 and newer, enabling a non-legacy portal on new workspaces loads default editable email templates.
 For existing non-legacy Portals, editable email templates must be loaded manually.
```

Comment:
Perhaps the wording here could be made more clear "For existing non-legacy Portals, editable email templates must be loaded manually." this only applies to 1.3.0.0
Or maybe it can just be removed as a note is in 
 Loading Email Templates on Existing Dev Portals	section

Reply:
stet for now; will revisit in a subsequent PR